_examples/nodeconfigs/simple.json

```diff
@@ -0,0 +1,20 @@
+{
+  "logger": {
+    "color": true,
+    "level": "info"
+  },
+  "nexus_name": "mynexus",
+  "tags": {
+    "foo": "bar"
+  },
+  "workload_types": [
+    {
+      "name": "CONFIGWORKLOAD",
```

Comment:
this isn't going to work. It will have to be something like `javascript` or `native` etc.

Reply:
This is just a "valid" config that will start the node in its current form....but will rename to native as that makes more sense